ui/src/app/cohort-search/modifier-page/modifier-page.component.tsx

```diff
@@ -204,418 +207,474 @@ interface Selection extends Criteria {
 }
 
 interface Props {
-  selections: Array<Selection>;
+  cdrVersionTiersResponse: CdrVersionTiersResponse;
   closeModifiers: (modifiers?: Array<Modifier>) => void;
-  workspace: WorkspaceData;
   cohortContext: any;
+  selections: Array<Selection>;
+  workspace: WorkspaceData;
 }
 
 export const ModifierPage = fp.flow(
+  withCdrVersions(),
   withCurrentWorkspace(),
   withCurrentCohortSearchContext()
-)(({ selections, closeModifiers, workspace, cohortContext }: Props) => {
-  const { ns, wsid } = useParams<MatchParams>();
-  const [calculateError, setCalculateError] = useState(false);
-  const [calculating, setCalculating] = useState(false);
-  const [count, setCount] = useState(null);
-  const [formErrors, setFormErrors] = useState([]);
-  const [formState, setFormState] = useState([...defaultFormState]);
-  const [formUntouched, setFormUntouched] = useState(false);
-  const [initialFormState, setInitialFormState] = useState(true);
-  const [loading, setLoading] = useState(true);
-  const [visitCounts, setVisitCounts] = useState(undefined);
+)(
+  ({
+    cdrVersionTiersResponse,
```

Comment:
Adding this prop changed the indentation of the whole component. So it looks like a lot of changes, but only a few lines actually changed.